app/questionnaire/navigation.py

```diff
@@ -50,8 +50,20 @@ def build_navigation(self, current_group_id, current_group_instance):
                 else:
                     logger.debug("building navigation", group_id=group['id'])
                     navigation.append(self._build_single_navigation(group, current_group_id, first_location))
+
+        if self._all_sections_completed(navigation):
```

Comment:
Instead of adding on a navigation item could we hide the final navigation (Summary or Confirmation) unless all sections are complete?